.github/workflows/NodeJS.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
```

Comment:
Can you set this trigger as well, please -> on: workflow_dispatch (Sam is trying on BE to do the same)